ivy/functional/backends/jax/gradients.py

```diff
@@ -74,8 +74,8 @@ def execute_with_gradients(
     /,
     *,
     retain_grads: bool = False,
-    xs_grad_idxs: Optional[Sequence[Sequence[Union[str, int]]]] = [[0]],
-    ret_grad_idxs: Optional[Sequence[Sequence[Union[str, int]]]] = [[0]],
+    xs_grad_idxs: Sequence[Sequence[Union[str, int]]] = ((0,),),
```

Comment:
could you please also update line [51](https://github.com/unifyai/ivy/pull/26746/files#diff-2ad043743db916943bf7613c8da8747238a2cd3b9682557ee9b30408360c097dR51) to `xs_grad_arr_idxs.append(list(grad_idx) + idx)`? After the `tuple` change, this change is also required. Thanks @Sai-Suraj-27 😄

Reply:
Done!

Reply:
thanks!